tests/test_configuration/test.ini

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
 [app:main]
 use = egg:kinto
 
+# Production settings
```

Comment:
Yeah I´m waiting on @leplatrem response about `kinto.collection_article_delete_enabled = false`  setting. If the documentation is not correct about those settings I think it´s ok to remove it from kinto.tpl, otherwise I can write a correct example, it´s just a i didn´t quite understand what´s incorrect.